### PR TITLE
{phys}[intel/2018a] BerkeleyGW v2.0.0

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.9, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/69b728bf92411e4884b3c0a904c0ce69 for a full test report.

File: easybuild/easyconfigs/b/BerkeleyGW/BerkeleyGW-2.0.0-intel-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+homepage = 'http://www.berkeleygw.org'
+description = """The BerkeleyGW Package is a set of computer codes that calculates the quasiparticle
+ properties and the optical responses of a large variety of materials from bulk periodic crystals to
+ nanostructures such as slabs, wires and molecules."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+toolchainopts = {'usempi': True, 'openmp': True}
+
+source_urls = ['https://berkeley.box.com/shared/static/']
+sources = [{'download_filename': 'wkqu37wu77tll53r8t1soe6ozqja67yy.gz', 'filename': SOURCE_TAR_GZ}]
```
Comment (boegel) [inline easybuild/easyconfigs/b/BerkeleyGW/BerkeleyGW-2.0.0-intel-2018a.eb]: @migueldiascosta this is the best option? :-/

File: easybuild/easyconfigs/b/BerkeleyGW/BerkeleyGW-2.0.0_fix_path.patch [hunk 2]
```diff
@@ -0,0 +1,11 @@
+--- Makefile.orig	2018-06-18 12:12:48.081168468 +0800
```
Comment (boegel) [inline easybuild/easyconfigs/b/BerkeleyGW/BerkeleyGW-2.0.0_fix_path.patch]: @migueldiascosta missing description + author (or upstream ref)

File: easybuild/easyconfigs/b/BerkeleyGW/BerkeleyGW-2.0.0-intel-2018a.eb (hunk 1 above)
Comment (migueldiascosta) [reply easybuild/easyconfigs/b/BerkeleyGW/BerkeleyGW-2.0.0-intel-2018a.eb]: @boegel afaik, it's the only option... :(

Comment (boegel) [reply easybuild/easyconfigs/b/BerkeleyGW/BerkeleyGW-2.0.0-intel-2018a.eb]: Maybe we should reach out to them and suggest to properly name their tarballs?

Mentioning `BGW-2.0.0.tar.gz` on the download page and then downloading a tarball that's named like gibberish doesn't make any sense...

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2524.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/d27cc73a57853e7a714180c9f18039d2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node3158.skitty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/c2f8f567c4d4526ba1b3e832444f5529 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2524.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/ab6b436de3417779298cb6da688567ef for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3158.skitty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/35f9ef4f1b1fbdb2fb6b0d69a4c4fbcb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2100.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6c2cc7c78ef076bc1c081feac5c02496 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @migueldiascosta!

**Pull request merged**